### ISSUE TITLE
Partial metadata for standard object files retrieved when using wildcard in package.xml to retrieve all CustomObject metadata

### ISSUE THREAD

**Issue by brosiusad** [joeferraro/MavensMate-SublimeText#168]
This appears to be related to https://github.com/joeferraro/MavensMate-SublimeText/issues/75.  And like that issue, I am trying to use MM/git with other devs that are using Force.com IDE.  I am using MM v2.2.0 (app v0.34).

When I right-click "refresh from server" for the "object" folder for standard objects like Case.object, or use "Clean Project", MM only retrieves <fields> for the Case.object file, but not <listViews>, <searchLayouts> and <actionOverrides> (Summer '13).  However, when I individually refresh Case.object, I get _all_ the metadata.  I tracked this down to the same problem as issue 75 above.  That is, package.xml had this:

```
<types>
    <members>*</members>
    <name>CustomObject</name>
</types>
```

and to fix the problem I needed to alter package.xml to look like this:

```
<types>
    <members>Account</members>
    <members>Asset</members>
    <members>Campaign</members>
    <members>CampaignMember</members>
    <members>Case</members>
    <members>Contact</members>
    <members>Contract</members>
    <members>Event</members>
    <members>Idea</members>
    <members>MyCustomObject__c</members>
    <members>Lead</members>
    <members>Opportunity</members>
    <members>OpportunityLineItem</members>
    <members>Task</members>
    <members>User</members>
    <name>CustomObject</name>
</types>
```

In other words, I have to list each and every object for which I want metadata.  Then when I issue a "Clean Project" or right-click and select "Refresh from Server..." on the "object" folder, it properly retrieves all the metadata for each .object file.  

I'm worried this might break going forward.  I don't want to have to update package.xml by hand!
